src/kubeseal_auto/debug.py

```
@@ -0,0 +1,90 @@
+            logger.debug("ic| %s:%d", caller.f_code.co_filename, caller.f_lineno)
+        return
+
+    expr = _get_call_expression()
+    if expr and len(args) == 1:
+        logger.debug("ic| %s: %r", expr, args[0])
+    else:
+        for arg in args:
+            logger.debug("ic| %r", arg)
+
+
+def configure_debug(enabled: bool) -> None:
```

Ruff (0.15.9)

[warning] Boolean-typed positional argument in function definition

(FBT001)